kbdcfg.lua

```diff
@@ -26,11 +26,9 @@ function kbdcfg.switch_by_name(name)
 end
 
 function kbdcfg.switch(layout)
-    for i, current in ipairs(kbdcfg.layouts) do
```

Comment:
Why did you remove this code? Are you sure that your code will work well in any cases?

Reply:
@echuraev no no, this is exactly what I was think about at today's morning :D I've just blindly replaced this part with the new one and it was working in my test case but definitely should break something. Give me some time I'll update the PR